_posts/2020-06-05-q.md

```diff
@@ -1 +1,10 @@
+---
+title: "This summer I'm looking forward to..."
+author: Q
+date: 2020-06-05
+---
 
+Going on vacation
+![vacation](https://www.familyvacationcritic.com/uploads/sites/19/2018/09/best-ideas-1280x640.jpg)
+
+Comment your reply here.
```

Comment:
But you could be here instead 
![Image of Cherry Blossoms](https://img.travelawaits.com/filter:centercrop/quill/7/8/6/9/b/2/7869b2f6d8e68e89909201dfcc4c65e30b105d5b.jpg?w=800&h=800)

Reply:
the dream: 
![japan](https://jw-webmagazine.com/wp-content/uploads/2017/05/jw-5d1b4ba8aed801.11236228.jpeg)